[func.wrap.func.con], [func.wrap.move.class], [func.wrap.copy.class], [any.class.general] Reword avoidance of 'dynamically allocated memory' as 'nesting an object within' (#7328)
cplusplus/draft: source/utilities.tex

The non-member \tcode{any_cast} functions provide type-safe access to the contained value.

\pnum

> Implementations should nest an object within\iref{intro.object}
> an object of type \tcode{any} for a small contained value.

Reviewer: Maybe it's better to say the following?
```suggestion
Implementations should make a small contained value
nested within\iref{intro.object} the \tcode{any} object.
```

\pnum
\recommended

> Implementations should nest an object within\iref{intro.object}
> a \tcode{function} for small callable objects,

Reviewer: ```suggestion
Implementations should make a small callable object
nested within\iref{intro.object} the \tcode{function} object,
```
(ditto below)

\pnum
\recommended

> Implementations should nest an object within\iref{intro.object}
> a \tcode{move_only_function} for a small contained value.

Reviewer: I believe we shouldn't say "contained value" here as it's not defined for `move_only_function`.
```suggestion
Implementations should make a small callable object
nested within\iref{intro.object} the \tcode{move_only_function} object.
```

\pnum
\recommended

> Implementations should nest an object within\iref{intro.object}
> a \tcode{copyable_function} for a small contained value.

Reviewer: ```suggestion
Implementations should make a small callable object
nested within\iref{intro.object} the \tcode{copyable_function} object.
```
